client/apps/edit/components/admin/test/index.coffee

```diff
@@ -64,7 +64,7 @@ describe 'EditAdmin', ->
       @view.featureFromInput('Artists')(
         currentTarget: $ "<input value='/andy-warhol' />"
       )
-      Backbone.sync.args[0][2].data.ids[0].should.equal 'andy-warhol'
+      Backbone.sync.args[0][1].id.should.equal 'andy-warhol'
 
```

Comment:
We aren't using the `ApiCollection` anymore for artist so I'm just testing against the request's `id` param